doc/dev/cephadm/developing-cephadm.rst

```diff
@@ -264,9 +264,15 @@ In order to setup Cephadm's box run::
 
 .. note:: It is recommended to run box with verbose (-v).
 
-After getting all needed images we can run::
+After getting all needed images we can create a simple cluster with::
 
-  sudo box -v cluster start --osds 3 --hosts 3
+  sudo box -v cluster start
+
+If you want to deploy the cluster with more osds and hosts::
+  # 3 osds and 3 hosts by default
+  sudo box -v cluster start --extended
+  # explicitly change number of hosts and osds
+  sudo box -v cluster start --extended --osds 5 --hosts 5
```

Comment:
What happens if I write `--osds 5 --hosts 5` without the `--extended` flag (BTW, shouldn't it be `--expanded`)? I'm just trying to understand the semantics of the flags.

Reply:
It won't make a change, I'll add a comment on that.